### PR TITLE
ACPF model implementation with unit tests

### PR DESCRIPTION
**Pull request by anyacastillo**

### CODE REVIEW

File: egret/models/acpf.py [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+    import os
+    from egret.parsers.matpower_parser import create_ModelData
+    from egret.models.acopf import create_psv_acopf_model, solve_acopf
+
+    path = os.path.dirname(__file__)
+    filename = 'pglib_opf_case14_ieee.m'
+    matpower_file = os.path.join(path, '../../download/pglib-opf-master/', filename)
+    md = create_ModelData(matpower_file)
+    kwargs = {'include_feasibility_slack':False}
+    md,m,results = solve_acopf(md, "ipopt", acopf_model_generator=create_psv_acopf_model,return_model=True, return_results=True,**kwargs)
```
Comment (michaelbynum) [inline egret/models/acpf.py:355]: Are `solve_acopf` and `create_psv_acopf_model` imported?

File: egret/models/tests/transmission_test_instances/acpf_solution_files/pglib_opf_case3012wp_k_acpf_solution.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (michaelbynum) [inline egret/models/tests/transmission_test_instances/acpf_solution_files/pglib_opf_case3012wp_k_acpf_solution.json:1]: I personally think we should avoid large problems in our automated testing, but this is okay for now.

**Review by michaelbynum — APPROVED** [review]

File: egret/models/tests/transmission_test_instances/acpf_solution_files/pglib_opf_case3012wp_k_acpf_solution.json (hunk 2 above)
Comment (carldlaird) [reply egret/models/tests/transmission_test_instances/acpf_solution_files/pglib_opf_case3012wp_k_acpf_solution.json:1]: I agree. This problem had some important features. I can look for other smaller problems that have similar features when I work on additional tests.

Comment (bknueven) [reply egret/models/tests/transmission_test_instances/acpf_solution_files/pglib_opf_case3012wp_k_acpf_solution.json:1]: The pglib-opf 300 bus case tends to be good for this, if you hadn't tried it already.

**Pull request merged**